packages/frontend/package.json

```diff
@@ -1,58 +1,57 @@
 {
-  "name": "@guardian/frontend-rendering",
-  "version": "0.1.0-alpha.1",
-  "description": "theguardian.com rendering tier",
-  "main": "index.js",
-  "license": "Apache-2.0",
-  "dependencies": {
-    "@emotion/core": "^10.0.5",
-    "@guardian/guui": "2.0.0-alpha.1",
-    "@guardian/pasteup": "2.0.0-alpha.2",
-    "@guardian/src-foundations": "^0.0.14",
-    "@types/ajv": "^1.0.0",
-    "@types/lodash.escape": "^4.0.6",
-    "@types/sanitize-html": "^1.18.3",
-    "ajv": "^6.10.1",
-    "aws-sdk": "^2.340.0",
-    "compose-function": "^3.0.3",
-    "compression": "^1.7.3",
-    "curlyquotes": "^1.3.2",
-    "dateformat": "^3.0.3",
-    "diskusage": "^0.2.5",
-    "dompurify": "^1.0.3",
-    "emotion": "^10.0.5",
-    "emotion-server": "^10.0.5",
-    "express": "^4.16.4",
-    "glob": "^7.1.3",
-    "html-minifier": "^3.5.14",
-    "jsdom": "^13.0.0",
-    "lodash.escape": "^4.0.1",
-    "lodash.get": "^4.4.2",
-    "minify-css-string": "^1.0.0",
-    "ophan-tracker-js": "^1.3.13",
-    "pm2": "^3.2.2",
-    "polished": "^1.9.2",
-    "raven-js": "3.19.1",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "sanitize-html": "^1.20.1"
-  },
-  "devDependencies": {
-    "@types/amphtml-validator": "^1.0.0",
-    "@types/compose-function": "^0.0.30",
-    "@types/compression": "^0.0.36",
-    "@types/dateformat": "^1.0.1",
-    "@types/dompurify": "^0.0.31",
-    "@types/express": "^4.16.0",
-    "@types/gapi.analytics": "^0.0.2",
-    "@types/google.analytics": "^0.0.39",
-    "@types/html-minifier": "^3.5.2",
-    "@types/jsdom": "^11.0.6",
-    "@types/lodash.get": "^4.4.4",
-    "@types/node": "^10.12.0",
-    "@types/raven-js": "^3.10.0",
-    "@types/react": "^16.4.11",
-    "@types/react-dom": "^16.0.7",
-    "amphtml-validator": "^1.0.23"
-  }
+    "name": "@guardian/frontend-rendering",
```

Comment:
Ah this file is annoying to edit as it gets reformatted automatically in a way that is different from the yarn/npm default.